Paintroid/res/values-ar/string.xml

```diff
@@ -108,7 +108,7 @@
   <string name="save_button_text">حفظ</string>
   <string name="discard_button_text">تجاهل</string>
   <string name="crop_algorithm_finish_text">انقر فوق قطع</string>
-  <string name="crop_nothing_to_corp">لا شيء للقطع</string>
+  <string name="crop_nothing_to_crop">لا شيء للقطع</string>
```

Comment:
please remove the string name changes from all string.xml files except the the default one. The strings have to be renamed via CrowedIn. @krytron please add these changes to CrowedIn.